config/README.md

```diff
@@ -10,17 +10,20 @@ The config file is named `huspot.config.yml`.
 
 There are a handful of standard config utils that anyone working in this library should be familiar with.
 
-#### getAndLoadConfigIfNeeded()
+#### getConfig()
 
-Locates, parses, and stores the `hubspot.config.yml` file in memory. This should be the first thing that you do if you plan to access any of the config file values. If the config has already been loaded, this function will simply return the already-parsed config values.
+Locates and parses the hubspot config file. This function will automatically find the correct config file. Typically, it defaults to the nearest config file by working up the direcotry tree. Custom config locations can be set using the following environment variables
 
-#### updateAccountConfig()
+- `USE_ENVIRONTMENT_CONFIG` - load config account from environment variables
+- `HUBSPOT_CONFIG_PATH` - specify a path to a specific config file
 
-Safely writes updated values to the `hubspot.config.yml` file. This will also refresh the in-memory values that have been stored for the targeted account.
+#### updateConfigAccount()
 
-#### getAccountConfig()
+Safely writes updated values to the `hubspot.config.yml` file.
 
-Returns config data for a specific account, given the account's ID.
+#### getConfigAccountById() and getConfigAccountByName()
+
+Returns config data for a specific account, given the account's ID or name.
 
 ## Example config
```

Comment:
I updated the existing content of the readme to reflect the changes I made, but it may make sense to rethink it completely

Reply:
Yeah good call. That could always be done as a follow up